nextflow_schema.json

```
@@ -982,28 +982,43 @@
                 "skip_damage_calculation": {
                     "type": "boolean",
                     "fa_icon": "fas fa-forward",
-                    "help_text": "Turns off the DamageProfiler module to compute DNA damage profiles."
+                    "help_text": "Turns off damage calculation to compute DNA damage profiles."
                 },
-                "damageprofiler_length": {
+                "damage_calculation_tool": {
+                    "type": "string",
+                    "default": "damageprofiler",
+                    "enum": ["damageprofiler", "mapdamage"],
+                    "fa_icon": "fas fa-tools",
+                    "description": "Specify the tool to use for damage calculation.",
+                    "help_text": "Specify the tool to be used for damage calculation. DamageProfiler is generally faster than mapDamage2, but the latter has an option to limit the number of reads used. This can significantly speed up the processing of very large files, where the damage estimates are already accurate after processing only a fraction of the input. Options: damageprofiler, mapdamage. By default, DamageProfiler is used."
```

```suggestion
                    "help_text": "Specify the tool to be used for damage calculation. DamageProfiler is generally faster than mapDamage2, but the latter has an option to limit the number of reads used. This can significantly speed up the processing of very large files, where the damage estimates are already accurate after processing only a fraction of the input."
```

Included in the default and enum sections :)